### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by nitin42**
Added some tests using nodejs `child` process.

### CODE REVIEW

**Comment by ThisIzKp** [comment]
PR related to #107, @nitin42 awesome job 👏

**Comment by siddharthkp** [comment]
fun fun fun!

In addition to this, do you think there's value in making these tests more granular/unit as well? (different PR)

Example: tests for `setup.js`

File: __tests__/index.test.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const test = require('ava')
+const execa = require('execa')
+
+test('Gives no errors when build size is less than the threshold value', t => {
+  return execa
+    .shell('bundlesize -f ./__tests__/dist/build.js -s 2KB')
```
Comment (vs1682) [inline __tests__/index.test.js]: I guess here bundlesize needs to be present globally, right?

Comment (nitin42) [reply __tests__/index.test.js]: May not be necessary because its just a sanity check and @siddharthkp wanted a demo 😄

**Comment by palashmon** [comment]
Closing this PR for now because it has not had recent activity. Thank you for your contributions.

**Pull request closed without merging**